app/utils/timezone.js

```diff
@@ -2,13 +2,18 @@
 // See LICENSE.txt for license information.
 
 import DeviceInfo from 'react-native-device-info';
+import moment from 'moment-timezone';
 
 import {isMinimumServerVersion} from 'mattermost-redux/utils/helpers';
 
 export function getDeviceTimezone() {
     return DeviceInfo.getTimezone();
 }
 
+export function getDeviceUtcOffset() {
+    return moment().utcOffset();
```

Comment:
is there a way to get this value without adding `moment` and `moment-timezone` as dependencies?

Reply:
There is this solution (although you might need to negate it since it returns 4 for UTC-4)

```
var date = new Date();
var offsetInHours = date.getTimezoneOffset() / 60;
```

https://stackoverflow.com/a/37553236

Reply:
yeah, I don't really like doing negating and assuming things personally and think there were mentions of it working slightly differently across platforms and simulator/emulator, I also prefer having the same library handling it in both webapp and mobile project and keeping it consistent, but we can switch it to that if you guys prefer, moment is not a large library, so not sure why it would be a concern